dashboard/lti_new.py

```diff
@@ -121,6 +121,11 @@ def extract_launch_variables_for_tool_use(request, message_launch):
     canvas_course_long_id = canvas_id_to_incremented_id(course_id)
     canvas_user_id = custom_params['canvas_user_id']
     canvas_user_long_id = canvas_id_to_incremented_id(canvas_user_id)
+    if 'email' not in launch_data.keys():
+        logger.error('Possibility that LTI launch by Instructor/admin becoming Canvas Test Student')
+        error_message = 'Student view is not available for My Learning Analytics.'
```

Comment:
Should we say "not yet available" since there was some discussion about making this available?

Reply:
I think stick with "not available" for the moment so we don't promise something that we don't know for sure will happen.